examples/lessons/1/index.js

```diff
@@ -1,5 +1,5 @@
 window.webGLStart = function() {
-  PhiloGL('lesson01-canvas', {
+  PhiloGL.PhiloGL('lesson01-canvas', {
```

Comment:
It should be possible to avoid this change to the examples (i.e. PhiloGL.PhiloGL) by modifying how we export the symbols from the library.

Not a big thing but we want to make as few mods to the examples as possible